sdk/containerregistry/Azure.ResourceManager.ContainerRegistry/api/Azure.ResourceManager.ContainerRegistry.net8.0.cs

```diff
@@ -3594,6 +3590,24 @@ protected virtual void JsonModelWriteCore(System.Text.Json.Utf8JsonWriter writer
         string System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.ContainerRegistry.Models.PipelineRunResult>.GetFormatFromOptions(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
         System.BinaryData System.ClientModel.Primitives.IPersistableModel<Azure.ResourceManager.ContainerRegistry.Models.PipelineRunResult>.Write(System.ClientModel.Primitives.ModelReaderWriterOptions options) { throw null; }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public readonly partial struct RoleAssignmentMode : System.IEquatable<Azure.ResourceManager.ContainerRegistry.Models.RoleAssignmentMode>
```

Comment:
I think the name of this has high possibility to collide with types in other RPs (such as authorization RP).
Could we rename this to `ContainerRegistryRoleAssignmentMode`? Or you could prepend the context where this thing is used
You could do this in `rename-mapping` in `autorest.md` file like this:
```
rename-mapping:
    RoleAssignmentMode: NewName
```
and then regenerate the library.
Please note, the name before the colon is its name in the swagger - please let me know if the above does not work

Reply:
Made requested changes